### PR TITLE
[ONE] Implement Jace, the Perfected Mind

### PR DESCRIPTION
**Pull request by AhmadYProjects**

### CODE REVIEW

**Comment by AhmadYProjects** [comment]
[[Jace, the Perfected Mind]]

File: Mage.Sets/src/mage/cards/j/JaceThePerfectedMind.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        this.addAbility(CompleatedAbility.getInstance());
+
+        // +1: Until your next turn, up to one target creature gets -3/-0.
+        Effect effect1 = new BoostTargetEffect(-3,0, Duration.UntilYourNextTurn);
+        effect1.setText("Until your next turn, up to one target creature gets -3/-0");
+        Ability ability1 = new LoyaltyAbility(effect1, 1);
+        ability1.addTarget(new TargetCreaturePermanent(0,1));
+        this.addAbility(ability1);
+        // -2: Target player mills three cards. Then if a graveyard has twenty or more cards in it, you draw three cards. Otherwise, you draw a card.
+        Effect effect2 = new JaceThePerfectedMindEffect();
```
Comment (theelk801) [inline Mage.Sets/src/mage/cards/j/JaceThePerfectedMind.java:45]: you don't need to declare the effect here as you're not using it

**Review by theelk801 — CHANGES_REQUESTED** [review]

File: Mage.Sets/src/mage/cards/j/JaceThePerfectedMind.java [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+import java.util.UUID;
+
+/**
+ *
+ * @author AhmadYProjects
+ */
+public final class JaceThePerfectedMind extends CardImpl {
+
+    public JaceThePerfectedMind(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.PLANESWALKER}, "{2}{U/P}{U}");
```
Comment (PurpleCrowbar) [inline Mage.Sets/src/mage/cards/j/JaceThePerfectedMind.java:29]: Build is failing because cost needs to be `"{2}{U}{U/P}"`

**Review by PurpleCrowbar — CHANGES_REQUESTED** [review]

File: Mage.Sets/src/mage/cards/j/JaceThePerfectedMind.java (hunk 2 above)
Comment (AhmadYProjects) [reply Mage.Sets/src/mage/cards/j/JaceThePerfectedMind.java:29]: I didnt touch the mana costs but ill change it

**Review by PurpleCrowbar — APPROVED** [review]

**Pull request merged**